### PR TITLE
Each application template is a library

### PR DESCRIPTION
**Pull request by arnifreyrs**
# Each application template is a library

## What

Build new libraries for all application templates, so each application template lib has all the specs for said application type.

## Why

This is to decrease bundle size, and also, when a developer is working on some **application A**, it should not need to trigger builds for other applications.

TODO - in a later PR I will upgrade all the README files for the application system.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Comment by sindrig** [comment]
LGTM but I'm not confident enough to review this :grimacing:

**Review by madebynoam — APPROVED** [review]
Lgtm!

File: workspace.json [hunk 1]
```diff
@@ -3098,6 +3098,122 @@
           }
         }
       }
+    },
```
Comment (magnearun) [inline workspace.json:3101]: We will need to move the extract strings script to each template lib

**Review by magnearun — CHANGES_REQUESTED** [review]
LGTM! 
Just a small fix to get the extract-strings script to work

File: workspace.json (hunk 1 above)
Comment (arnifreyrs) [reply workspace.json:3101]: done

**Review by eirikurn — APPROVED** [review]

**Review by magnearun — APPROVED** [review]

**Pull request merged**